docs/linter/eslint-plugin/rules/json-dependency-versions-harmonize.md

```diff
@@ -0,0 +1,99 @@
+# Verify that the dependencies are aligned in the monorepos sub packages
+
+This rule verify, for each `package.json`, if the dependencies are aligned with the ones defined in the other `package.json` of the workspace.
+
+## Prerequisite
+
+The linter is dedicated to JSON files and requires the setup of the [jsonc-eslint-parser](https://www.npmjs.com/package/jsonc-eslint-parser).
+
+Example of a ESLint configuration:
+
+```json
+{
+  "parser": "jsonc-eslint-parser",
+  "plugins": [
+    "@o3r"
+  ],
+  "extends": [
+    "@o3r:recommended-json"
+  ]
+}
+```
+
+## Options
+
+### alignPeerDependencies
+
+Enforce to align the version of the dependencies with the latest range.
+
+**Default**: *false*
+
+**Usage**:
+
+```json
+{
+  "@o3r/json-dependency-versions-harmonize": [
+    "error",
+    {
+      "alignPeerDependencies": true
+    }
+  ]
+}
+```
+
+### dependencyTypes
+
+List of dependency types to update
+
+**Default**: ['optionalDependencies', 'dependencies', 'devDependencies', 'peerDependencies', 'generatorDependencies']*
```

Comment:
I just move the existing script but to reply, I would say "yes".
If it does not exist in the targeted package.json it will not do anything in any case.
(to compare, Renovate is based on the regex `/dependencies$/i`, so we are still less permissive)